debezium-core/src/main/java/io/debezium/transforms/outbox/EventRouter.java

```diff
@@ -68,7 +69,16 @@
     public R apply(R r) {
         // Ignoring tombstones
         if (r.value() == null) {
-            LOGGER.info("Tombstone message {} ignored", r.key());
+            LOGGER.info("Tombstone message ignored. Message key: \"{}\"", r.key());
+            return null;
+        }
+
+        // Ignoring messages which do not adhere to the CDC Envelope, for instance:
+        // Heartbeat and Schema Change messages
+        if (r.valueSchema() == null ||
+                r.valueSchema().name() == null ||
+                !r.valueSchema().name().endsWith(RECORD_ENVELOPE_SCHEMA_NAME_SUFFIX)) {
+            LOGGER.warn("Message without Debezium CDC Envelope ignored. Message key: \"{}\"", r.key());
```

Comment:
@gunnarmorling would this fit better as info than wan?

Reply:
Yes, it shouldn't be warn, as it's an expected operation. In fact, probably best to move it to debug, as putting it to info may swamp to log with a regular thing happening.

Reply:
Switched to info, is that ok?

Reply:
LOL, we replied at the same time :) I think it should be debug actually; same for the other one above (already existing before).

Reply:
Sure, I can do that :)

I'd actually in the future like to make it easier to configure, I'm a bit of control freak and I actually monitor those (in sentry ;) )

Reply:
Done b tw

Reply:
> I'd actually in the future like to make it easier to configure

How would you envision this? I'm all for improving operational aspects where we can.

Reply:
I have to learn a bit more on what is better in the ops point of view, log or exception for instance if we can add context to it or play with log4j configuration. But I also don't know since we're bound to the connect framework, maybe we can chat later so you can give me some directions!

Reply:
Increasing the log level via the log4j config is the standard way of obtaining more info, I'd say. Open for alternatives and additions that help to gain more insight, e.g. we were thinking about adding support for OpenTracing and other things.